src/select/item-list/SelectItemList.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {forwardRef, Ref} from "react";
+import classNames from "classnames";
+
+import {Option} from "../util/selectTypes";
+import List, {ListElementType} from "../../list/List";
+import Select from "../Select";
+import {ListProps} from "../..";
+
+export type SelectItemListProps<T extends Option = Option> = {
+  options: T[];
+  customClassName?: string;
+  contentRenderer: (option: T) => React.ReactNode;
```

Comment:
Shall we change the order? It is more readable imo.

```suggestion
  contentRenderer: (option: T) => React.ReactNode;
  customClassName?: string;
```